Refresh and refocus the Hells (#2064)
crawl/crawl: crawl-ref/source/dat/descript/monsters.txt

<demonspawn>
%%%%
creeping inferno

A formless mass of ancient fire, bound by heat and malevolence. The slightest
disturbance will destroy the bonds holding it together and unleash a fire

> storm. It longs for that release yet it seems somehow reluctant to move while
> being observed.

Reviewer: ```suggestion
storm. It longs for that release, yet seems reluctant to move while observed.
```

%%%%
crimson imp

A small, ugly minor demon. Its fanged maw seems locked in a permanent malign
grin, and its horns protrude obscenely from its scaly pate.
%%%%
crystal echidna

> A spiny, long snouted rodent grown out of pure living crystal. It is

Reviewer: ...should we have some explanation for why a rodent is lurking in hell, or are we just going to roll with it?

Author: I added `serial_glass` to Dis except it places green crystal instead of glass. Might also add green crystal to some of the layouts. Beyond the terrain "explanation" I don't think we need to say more: it's a city, it has vermin!

that removed it from its home. These horrors are often the result of mistaken
summoning spells and have slain many aspiring wizards.
%%%%
nargun

An ancient stone creature that has lived among these frozen caves since before

> Antaeus rent Cocytus into being. It tolerates its new neighbors begrudgingly,

Reviewer: Yes! Lore! Loving it

A ravenous and incredibly buggy monster. Please report its existence to the
DevTeam.
%%%%
putrid mouth

> A starved spirit whose mouth is a pit of rot and emits vile fumes. Its putrid

Reviewer: ```suggestion
A starved spirit whose mouth is a pit of rot, emitting vile fumes. Its putrid
```

maw cannot eat the food it finds, so it moans with rage and desparate hunger.

> The filth on its claws is withering.

Reviewer: Is this last sentence mechanically significant? I wonder if we can cut it, since we describe monster AFs in the rest of xv anyway.

skin shines like molten mercury, magical energies arc from its pointed snout,
and it moves with terrifying swiftness.
%%%%
quicksilver elemental

> A mercurial spirit drawn from the elemental plane of earth, which exists in
> this world by inhabiting a shimmering body of liquid metal.

Reviewer: Let's avoid run-ons:

```suggestion
A mercurial spirit drawn from the elemental plane of earth. It exists in
this world by inhabiting a shimmering body of liquid metal.
```

Also, huh, really pulling in the D&D/nethackisms for this one, huh? Not sure how I feel about an 'elemental plane of earth' suddenly entering the cosmology... suppose it doesn't do any harm.

Author: Earth elementals are already from there, so are our other elementals!

Reviewer: Wow, i'd totally forgotten that! Need to brush up on my lore, clearly :)

A brightly coloured snake, very comfortable in the water. It has a highly
potent venomous bite.
%%%%
searing wretch

> An animated corpse alight with terrible fire. The flames are fueld by its

Reviewer: ```suggestion
An animated corpse alight with terrible fire. The flames are fueled by its
```

regenerating flesh, which bubbles and melts as it grows. Its bite leaves

> its victims covered in flaming gore.

Reviewer: Same comment here about the last sentence being probably redundant with AF_ descriptions.

Author: I wanted to specify why the sticky flame was sticking, but maybe it's not needed.

A relatively small grey dragon, with a resilient-looking scaly hide and steam
pouring from its mouth.
%%%%
stoker

> A blacksmith who made an unholy pact in life, now chained to tend the fires of

Reviewer: It seems weird that so many blacksmiths made unholy pacts. Maybe just 'a mortal'?

A small but greatly-feared spider whose enchanted bite causes victims to break
into a frenzied dance.
%%%%
tainted leviathan

Before even the oldest of ancient liches mastered life and death, a group of
titans attempted to pull themselves free of mortality through brute strength.
They almost succeeded before rot dragged them to Ereshkigal. A glimmer of their

> former beauty remains, alluring despite the stench.

Reviewer: Sick.

Wherever they flow, so may she, and ponds and rivers surge and leap at her
whim.
%%%%
wendigo

A gaunt spirit of frozen hunger, so ravenous it will consume friend and foe

> alike. According to legend if one cuts off its legs it will feast upon its own
> marrow. It stalks those lost wandering in the cold and hunts with the force of

Reviewer: ```suggestion
alike. According to legend, if one cuts off its legs, it will feast upon its own
marrow. It stalks those lost wandering in the cold and hunts with the force of
```
